tests/integration/targets/eos_bgp_global/vars/main.yaml

```diff
@@ -37,6 +37,7 @@ merged:
     neighbor:
       - maximum_received_routes:
           count: 12000
+        neighbor_address: "peer1"
```

Comment:
Tests would need updating based on above comment. But in general, to make sure that both the keys work when used in a playbook, please have a task that uses `peer` as well. Having it in unit test also works.

Reply:
Both the keys are present in the integration testcases.